src/main/resources/samples.conf

```
@@ -108,7 +80,7 @@ dataObjects {
     table {
       db = default
       name = "ab_hive"
-      primary-keys = [id]
+      primary-key = [id]
```

Isn't it primaryKey now?

```suggestion
      primaryKey = [id]
```

---

works with CamelCase and dash :-)